Feature/MapBoxGPS

yairme opened the pull request:
Refactor the CameraControl script by removing the unused System namespace. Also, refactor the DevicePosition script by renaming variables and improving code readability.

Note: This message does not include any meta information like issue references, tags, or author names.

stefan-hol committed:
GPS Update

Made sure to get the latitude fortitude and longitude. A compass facing north. And a way to find out if it is running on android or IOS

Commit:
chore: Remove unused scripts and meta files

Commit:
Merge branch 'main' into feature/MapBoxGPS

Commit:
chore: Update InteractableObject to use Mapbox GPS coordinates

Commit:
Added Interactable object:
Player Detection.

Added SpawnWayPoints:
Script to spawn the InteractableObject, also to assign the events.

Added GeoCoordinate:
Imported script to get the location of a coordinates and translate it to Vectors for Unity.

ToDo: Created a Compass.

Commit:
Reverted touch controls to before version

Small updates to cameraControl

Commit:
chore: Remove unused scripts and meta files

Commit:
Revert "chore: Remove unused scripts and meta files"

This reverts commit 858141cf636e48fe894b969ef38d4b1810b9b646.

Commit:
feat: Add InteractableObject and GeoCoordinate scripts

The InteractableObject script is added to handle player detection and spawn waypoints. The GeoCoordinate script is imported to get the location of coordinates and translate them to vectors for Unity. Also, a compass is created as a todo item.

Note: This message does not include any meta information like issue references, tags, or author names.

Commit:
chore: Remove unused scripts and meta files

Commit:
chore: Refactor CameraControl and DevicePosition scripts

Refactor the CameraControl script by removing the unused System namespace. Also, refactor the DevicePosition script by renaming variables and improving code readability.

Note: This message does not include any meta information like issue references, tags, or author names.

yairmegaming committed:
Merge branch 'main' into feature/MapBoxGPS

stefan-hol reviewed (changes requested):
make it true to the code convention

yairmegaming committed:
Cleaned up scripts to have code convetion

yairmegaming committed:
refactor: Update location provider variables in DeviceRotation and PlayerLocation scripts

Update the variable names for the location provider in the DeviceRotation and PlayerLocation scripts to improve code clarity and consistency.

Note: This message does not include any meta information like issue references, tags, or author names.

yairmegaming committed:
refactor: Update variable names for location provider in PlayerLocation script

yairmegaming committed:
Added function to add sprites to the gameObject
